HARK/distribution.py

```diff
@@ -411,22 +446,69 @@ def approx(self, N, tail_N=0, tail_bound=None, tail_order=np.e):
                     atoms[i] = (
                         -0.5
                         * np.exp(self.mu + (self.sigma**2) * 0.5)
-                        * (erf(tempTop) - erf(tempBot))
+                        * (math.erf(tempTop) - math.erf(tempBot))
                         / pmv[i]
                     )
                 else:
                     atoms[i] = (
                         -0.5
                         * np.exp(self.mu + (self.sigma**2) * 0.5)
-                        * (erfc(tempBot) - erfc(tempTop))
+                        * (math.erfc(tempBot) - math.erfc(tempTop))
                         / pmv[i]
                     )
 
         else:
             pmv = np.ones(N) / N
             atoms = np.exp(self.mu) * np.ones(N)
+
+        limit = {
+            "dist": self,
+            "method": "equiprobable",
+            "N": N,
+            "endpoints": endpoints,
+            "tail_N": tail_N,
+            "tail_bound": tail_bound,
+            "tail_order": tail_order,
+        }
```

Comment:
This may be implemented a little more generically using `**kwargs` to capture the arguments given the `discretize` method.

https://www.digitalocean.com/community/tutorials/how-to-use-args-and-kwargs-in-python-3

Reply:
That's a good point. Will think about this and leave it for a different PR.